nntrainer/include/bn_layer.h

```diff
@@ -125,6 +125,7 @@ class BatchNormalizationLayer : public Layer {
   Tensor var;
   Tensor gamma;
   Tensor beta;
+  Tensor x_hat;
```

Comment:
Add description on this variable or rename `x_hat` to `x_normalized`

Reply:
done.